src/lib/site-import/db-file-import.js

```diff
@@ -0,0 +1,39 @@
+/**
+ * @flow
+ * @format
+ */
+
+/**
+ * External dependencies
+ */
+
+export async function importFile() {}
+
+export function currentUserCanImportForApp( app: any ): boolean {
+	// TODO: implement
+	return !! app;
+}
+
+export function isSupportedApp( app: any ): boolean {
+	// TODO: implement
```

Comment:
Here is a spot I could use a hand implementing

Reply:
We are just checking here that this is a WordPress site, or are we checking for a feature flag as well?

Reply:
@dchymko This function is just for a WordPress-type site (so WP preprod & prod app types).

There are probably other constraints imposed by the back-end that we'll want to catch in this block as well (some subsite-specific things, presence of lock meta, etc.), but those can be added separately.

RE: feature flags, I'm not yet familiar with how we do those in this project.  We'll want to add that in too if it's not a huge lift.

Thanks for taking a look!